images/ami/rhel-82.yaml

```diff
@@ -3,12 +3,14 @@ download_images: true
 packer:
   ami_filter_name: "RHEL-8.2.0*2020*-x86_64-*"
   ami_filter_owners: "309956199498"
+  ami_filter_name: 
   distribution: "RHEL"
   distribution_version: "8.2"
   source_ami: ""
   ssh_username: "ec2-user"
   root_device_name: "/dev/sda1"
   volume_size: "15"
+  remote_folder: "/home/ec2-user"
```

Comment:
Is this change needed? What about all the other images?

Reply:
we decided to keep this change only for RHEL-8.2 yaml to make sure at least one e2e test covers it. we didnt made this change to all other images since this parameter is optional. 
Do you have any preference on removing this from here or making the same change for all the AMIs?

Reply:
Since these are visible to end users can we move this config to our RHEL 8.2 e2e config in TC instead?

Reply:
That works too. we have to test if KIB merges `packer:` block properly when provided as overrides. I will give it a try and move to TC config if works.

Reply:
Tested and changed TC configuration of `E2E RHEL 8.2 Offline FIPS` build